Documentation/malaria_documentation_summary .md

```diff
@@ -0,0 +1,222 @@
+<h1>Detection of Malaria 🦟🦠</h1>
+
+<h3>INTRODUCTION</h3>
+
+<p>Malaria is a life-threatening disease. It’s typically transmitted through the bite of an infected <b>anopheles mosquito</b>. Infected mosquitoes carry the <b>plasmodium parasite</b>. When this mosquito bites you, the parasite is released into your bloodstream. It is considered as an endemic in many parts of the world.</p></n><p>Malaria detection is performed by examining a drop of the patient’s blood, spread out as a “blood smear” (red blood cell) on a microscope slide. This blog focuses on improving malaria detection from such patches segmented from the microscopic images of blood smears by introducing a deep convolutional neural network. Compared to the traditional methods that use tedious hand engineering feature extraction, the proposed method uses deep learning in an end-to-end arrangement that performs both feature extraction and classification directly from the raw segmented patches of the red blood smears.</p>
+
+<h3>RISK FACTOR</h3>
+
+<p>Malaria is commonly associated with poverty and has a major negative effect on economic development. In Africa, it is estimated to result in losses of <b>US$12 billion</b> a year due to increased healthcare costs, lost ability to work, and negative effects on tourism</p>
```

Comment:
We need to add the article here
```
</b> a year due to increased healthcare costs, lost the ability to work, and negative effects on tourism</p>
```

Reply:
@bhadreshpsavani noted, will make changes.